Bonfire-Caesars-Cipher.md

```diff
@@ -48,9 +48,33 @@ function rot13(str) {
 }
 ```
 
+#### Second solution (advanced)
+
+```js
+function rot13(str) {
+  // Split str into a character array
+  return str.split('')
+  // Iterate over each character in the array
+    .map.call(str, function(char) {
+      // Convert char to a character code
+      x = char.charCodeAt(0);
+      // Checks if character lies between A-Z
+      if (x < 65 || x > 91) {
+        return String.fromCharCode(x);  // Return un-converted character
+      }
+      // If the ASCII gets out of range add 26 to it. (ASCII+26-13)
```

Comment:
This is confusing - you say "add 26 to it", but it's really adding 13?   
Maybe something like:
> N = ASCII 78, if the letter is less than 78, shift forward 13 places

Or something like that?